deploy/olm-catalog/infinispan-operator.clusterserviceversion.yaml

```diff
@@ -255,7 +257,7 @@ spec:
                 - watch
                 - create
                 - update
-                - delete
```

Comment:
Won't this cause issues when deleting Infinispans or is `delete` operation included in `patch`?

Reply:
@Crumby seems to be not, as operator itself never deletes own CR's

Reply:
yeah, atm is not needed. I've just aligned csv with the role.yaml and clusterrole.yaml file. I'm going to add some code to the codegen target that automatize this step